sycl/test/basic_tests/interop-level-zero-2020.cpp

```diff
@@ -0,0 +1,110 @@
+// REQUIRES: level_zero_dev_kit
```

Comment:
Does this test even run? I don't see `level_zero_dev_kit` to be set in https://github.com/intel/llvm/blob/sycl/sycl/test/lit.cfg.py

Reply:
We certainly should be able to run this since we download/build L0 dev kit to build L0 plugin.
How to see if the test was actually run? If it wasn't how to adjust lit testing such that L0 dev kit is seen?

Reply:
I usually just intentionally break test to see if it fails.
To make in-tree lit support this feature, you probably need to do something similar to llvm-test-suite's lit.cfg.py: https://github.com/intel/llvm-test-suite/blob/be09e0c6e49cad52f4dfbfa29d693f334c07c1d1/SYCL/lit.cfg.py#L124-L129

Reply:
You were right, the test is skipped as unsupported. It does pass if I just remove the REQUIRES: level_zero_dev_kit
Should I really be checking for L0 includes like in llvm-test-suite, or just unconditionally add this extension in in-tree lit (since L0 is always downloaded and built)?

Reply:
> Should I really be checking for L0 includes like in llvm-test-suite, or just unconditionally add this extension in in-tree lit (since L0 is always downloaded and built)?

I'd prefer the latter, but we can clarify it with @vladimirlaz

Reply:
@vladimirlaz suggested to just removes REQUIRES in case it is always available.